great_expectations/data_asset/data_asset.py

```diff
@@ -934,15 +934,12 @@ def validate(
 
             # Warn if our version is different from the version in the configuration
             try:
-                if (
-                    expectation_suite.meta["great_expectations.__version__"]
-                    != ge_version
-                ):
+                if expectation_suite.meta["great_expectations_version"] != ge_version:
```

Comment:
Mark with a todo to return and fix later, but for now let's accept both versions so this isn't breaking.

Reply:
Hi @jcampbell Sorry, I'm not clear if I should change stuff now? Should I just revert these changes at the specified locations and add a comment?

Reply:
Just came across this :) I think this meant allow both `great_expectations_version` and `great_expectations.__version__` to make the change non-breaking? And add a TODO to clean up later.